usabilla-react-native/src/index.js

```diff
@@ -36,11 +36,20 @@ function setFormDidClose(callback) {
     }
 }
 
+function setCustomVariables(customVariables) {
+    UsabillaBridge.setCustomVariables(customVariables, (errorMessage) => console.log(errorMessage))
```

Comment:
Why there is an errorMessage callback in this setter ? we don't send an error response from Native SDKs now, so we should only send the dict object.